examples/advanced/lr-newton-raphson/README.md

```diff
@@ -0,0 +1,243 @@
+  which relies on the
+  [`ModelPersistor`](https://nvflare.readthedocs.io/en/main/glossary.html#persistor). A
+  custom
+  [`NewtonRaphsonModelPersistor`](job/newton_raphson/app/custom/newton_raphson_persistor.py)
+  is implemented in this example, which is based on the
+  [`NPModelPersistor`](https://github.com/NVIDIA/NVFlare/blob/main/nvflare/app_common/np/np_model_persistor.py)
+  for numpy data, since the _model_ in the case of logistic regression
+  is just the parameter vector $\theta$ that can be represented by a
+  numpy array. Only the `__init__` method needs to be re-implemented
+  to provide a proper initialization for the global parameter vector
+  $\theta$.
+- During each training round, the global model will be sent to the
+  list of participating clients to perform a training task. This is
+  done using the
+  [`send_model()`](https://github.com/NVIDIA/NVFlare/blob/d6827bca96d332adb3402ceceb4b67e876146067/nvflare/app_common/workflows/model_controller.py#L99)
+  method under the hood, from the `ModelController` base class. Once
+  the clients finish their local training, results will be collected
+  and sent back to server as
+  [`FLModel`](https://nvflare.readthedocs.io/en/main/programming_guide/fl_model.html#flmodel)s.
+- Results sent by clients contain their locally computed gradient and
+  Hessian. A [custom aggregation
+  function](job/newton_raphson/app/custom/newton_raphson_workflow.py#L111)
+  is implemented to get the averaged gradient and Hessian, and compute
+  the Newton-Raphson update for the global parameter vector $\theta$,
+  based on the theoretical formula shown above. The averaging of
+  gradient and Hessian is based on the
+  [`WeightedAggregationHelper`](https://github.com/NVIDIA/NVFlare/blob/fa4d00f76848fe4eb356dcde417c136047eeab36/nvflare/app_common/aggregators/weighted_aggregation_helper.py#L20),
+  which weighs the contribution from each client based on the number
+  of local training samples. The aggregated Newton-Raphson update is
+  returned as an `FLModel`.
+- After getting the aggregated Newton-Raphson update, an
+  [`update_model()`](job/newton_raphson/app/custom/newton_raphson_workflow.py#L172)
+  method is implemented to actually apply the Newton-Raphson update to
+  the global model.
+- The last step is to save the updated global model, again through
+  the `NewtonRaphsonModelPersistor`.
+
+
+On the client side, the local training logic is implemented
+[here](job/newton_raphson/app/custom/newton_raphson_train.py). The
+implementation is based on the [`Client
+API`](https://nvflare.readthedocs.io/en/main/programming_guide/execution_api_type.html#client-api). This
+allows user to add minimum `nvflare`-specific codes to turn a typical
+centralized training script to a federated client side local training
+script.
+- During local training, each client receives a copy of the global
+  model, sent by the server, using `flare.receive()` API. The recieved
```

Comment:
Few misspellings of received. See below also.

Reply:
Fixed, thanks @holgerroth